text/0005-white-flag/0005-white-flag.md

```diff
@@ -0,0 +1,170 @@
++ Feature name: `white-flag`
++ Start date: 2020-03-06
++ RFC PR: [iotaledger/protocol-rfcs#0005](https://github.com/iotaledger/protocol-rfcs/pull/5)
+
+# Summary
+
+This RFC is part of a set of protocol changes, [Chrysalis](https://roadmap.iota.org/chrysalis), aiming at improving the
+network before [Coordicide](https://coordicide.iota.org/) is complete.
+
+The feature presented in this RFC, White Flag, allows milestones to confirm conflicting bundles by enforcing
+deterministic ordering of the Tangle and applying only the first bundle(s) that does not violate the ledger state.
+
+The content of this RFC is based on [Conflict white flag: Mitigate conflict spamming by ignoring conflicts](https://iota.cafe/t/conflict-white-flag-mitigate-conflict-spamming-by-ignoring-conflicts/233).
+
+# Motivation
+
+- Eliminates the [Conflict spamming attack](https://iota.cafe/t/conflict-spamming-attack/232);
+- As conflicts are ignored in the balance computation, they don’t need to be considered during tip selection of the
+nodes allowing much easier tip selection algorithms leading to increased TPS;
+- By using this approach in combination with an appropriate TSA, during regular use, no honest transaction will ever
+require re-attaching leading to increased CTPS;
+- Doesn't come with added computation complexity by integrating nicely into already existing algorithms;
+
+# Detailed design
+
+First, let's define what it means for a bundle to be:
+
+- referenced (indirectly or not) by another bundle: it is in the past cone of this bundle;
+- confirmed: it is referenced by a milestone;
+- applied: it is confirmed and applied to the ledger state;
+- ignored: it is confirmed but not applied;
+- conflicting: it would lead to an invalid ledger state if applied;
```

Comment:
```suggestion
First, let's define what it means for a bundle A to be:

- referenced (indirectly or directly) by bundle B: A is contained in the past cone of B;
- confirmed: A is referenced by a milestone;
- applied: A is confirmed and applied to the ledger state;
- ignored: A is confirmed but not applied;
- conflicting: A would lead to an invalid ledger state if applied;
```
I agree with @Alex6323, it think the above would be a tiny bit clearer.